### PR TITLE
Implement private registry test

### LINKED ISSUE #2140: Get a base private registry running in VIC
We need a private registry running on VIC for use with our CI system.  We need one that allows pushing of images on it.  An idea is to use a DIND container with a registry.  If that doesn't work, find out why.

### PR DESCRIPTION
**Pull request by mhagen-vmware**
This test has a number of issues being run within the integration environment, so moving it to manual test cases for now.

Fixes #2140

### CODE REVIEW

**Comment by rajanashok** [comment]
LGTM

**Comment by anchal-agrawal** [comment]
LGTM :+1:

File: tests/manual-test-cases/Group5-Functional-Tests/5-9-Private-Registry.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#Test Steps:
+1. Install a new VCH appliance into the vSphere Server
+2. Start the docker registry locally:  
+```docker run -d -p 5000:5000 --name registry registry```
+3. Pull an image and tag it for the new local registry:  
+```docker tag busybox localhost:5000/busybox:latest```
+4. Push the tagged image to the registry:  
+```docker push localhost:5000/busybox```
+5. Attempt to pull the local registry image using the VCH appliance:  
+```docker pull ${params} 172.17.0.1:5000/busybox```
```
Comment (matthewavery) [inline tests/manual-test-cases/Group5-Functional-Tests/5-9-Private-Registry.md]: should this be `localhost:5000`? The other registry operations work on that range.

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-9-Private-Registry.md]: No, that last call is switching from the localhost calls to the VCH, that is why ${params} showed up too in that last call.

Comment (matthewavery) [inline tests/manual-test-cases/Group5-Functional-Tests/5-9-Private-Registry.md]: alright, just making sure. I saw the params so figured that was actually targeting a VCH. Was a little wary seeing an IP there.

**Comment by matthewavery** [comment]
LGTM

**Pull request merged**